lib/internal/navigator.js

```
@@ -0,0 +1,36 @@
+'use strict';
+
+const {
+  ObjectDefineProperties,
+} = primordials;
+
+const {
+  kEnumerableProperty,
+} = require('internal/util');
+
+const {
+  getOSInformation,
+} = internalBinding('os');
+
+class Navigator {
```

new Navigator() should throw an error (or in this case, `new navigator.constructor()`).

---

adding on, in browsers both `navigator` and `Navigator` are exposed. Is there a reason to only expose one?

---

I addressed your concerns. Appreciate reviewing again.